src/unittests/maths.spec.ts

```diff
@@ -842,3 +849,241 @@ g.test('test,math,correctlyRoundedFlushToZeroOnly')
       `correctlyRounded(${test_val}, ${target}) returned ${got}. Expected ${is_correct}`
     );
   });
+
+interface lerpCase {
+  min: number;
+  max: number;
+  t: number;
+  result: number;
+}
+
+g.test('test,math,lerp')
+  .paramsSimple<lerpCase>([
+    // Infinite Bounds
+    { min: 0.0, max: Number.POSITIVE_INFINITY, t: 0.5, result: Number.NaN },
+    { min: Number.NEGATIVE_INFINITY, max: 1.0, t: 0.5, result: Number.NaN },
+    { min: Number.NEGATIVE_INFINITY, max: Number.POSITIVE_INFINITY, t: 0.5, result: Number.NaN },
+
+    // [0.0, 1.0] cases
+    { min: 0.0, max: 1.0, t: Number.NEGATIVE_INFINITY, result: 0.0 },
```

Comment:
Ohhh, this is unexpected. Most `lerp` implementations do not also clamp. I'd either remove the clamping, or rename to `clampedLerp` or something.

Reply:
Done, converted to be not clamp. Also removed the requirement for the params being ordered.